app/models/spotlight/resources/iiif_manifest.rb

```diff
@@ -36,6 +39,16 @@ def add_document_id
         solr_hash[exhibit.blacklight_config.document_model.unique_key.to_sym] = compound_id
       end
 
+      def add_parent_id
+        if parent
+          solr_hash[parent_id_field] = [parent.to_solr[exhibit.blacklight_config.document_model.unique_key.to_sym]]
```

Comment:
Maybe we should expose `parent.to_solr[exhibit.blacklight_config.document_model.unique_key.to_sym]` through a nice helper, so we can skip unnecessarily creating the solr document?